include/k4arecord/record.hpp

```diff
@@ -102,11 +130,11 @@ class record
      *
      * \sa k4a_record_add_tag
      */
-    void add_tag(const char *name, const char *value)
+    void add_tag(const char *name, const std::string &value)
```

Comment:
I'm not sure I understand the rationale behind this change.  I think this just makes it so you have to wrap a C string in a STL string, but we're not actually taking ownership of that string, are we?  I don't object to adding an overload to take a std::string, but I think we probably want to keep the C-string overload, no?

Also it seems a little weird that we're mixing and matching C and C++ strings here